Defining a version for __hurd_threadvar_stack_offset in both libc.so and ld.so
seems to lead to a conflict.

Index: eglibc-2.18/sysdeps/mach/hurd/Versions
===================================================================
--- eglibc-2.18.orig/sysdeps/mach/hurd/Versions	2013-10-10 04:45:12.000000000 +0000
+++ eglibc-2.18/sysdeps/mach/hurd/Versions	2013-10-10 04:45:50.000000000 +0000
@@ -16,7 +16,7 @@
     # variables that must be shared with libc
     __hurd_sigthread_stack_base; __hurd_sigthread_stack_end;
     __hurd_sigthread_variables;
-    __hurd_threadvar_stack_mask;  __hurd_threadvar_stack_offset;
+    __hurd_threadvar_stack_mask;
 
     # functions that must be shared with libc
     __close; __getcwd; __getpid;
